src/components/Restoration/RestorationType.vue

```diff
@@ -20,7 +20,7 @@
         >
           <v-radio-group
             v-model="selectRestorationType"
-            class="mt-0 pt-0"
+            class="mt-0 mb-n5 pt-0"
             @change="changeRestorationType()"
           >
```

Comment:
In general, we shouldn't "shorten" components like this unless there is no other way.

In this case, there may be a couple of ways to shorten this radio group:
- make it dense
- hide details if there aren't any (auto mode)

Ref: https://v2.vuetifyjs.com/en/api/v-radio-group/#props